tests/e2e/configurer/chain/node.go

```diff
@@ -92,22 +92,20 @@ func (n *NodeConfig) Stop() error {
 
 // WaitUntil waits until node reaches doneCondition. Return nil
 // if reached, error otherwise.
-func (n *NodeConfig) WaitUntil(doneCondition func(syncInfo coretypes.SyncInfo) bool) error {
+func (n *NodeConfig) WaitUntil(doneCondition func(syncInfo coretypes.SyncInfo) bool) {
 	var latestBlockHeight int64
 	for i := 0; i < waitUntilrepeatMax; i++ {
 		status, err := n.rpcClient.Status(context.Background())
-		if err != nil {
-			return err
-		}
+		require.NoError(n.t, err)
 		latestBlockHeight = status.SyncInfo.LatestBlockHeight
 		// let the node produce a few blocks
 		if !doneCondition(status.SyncInfo) {
 			time.Sleep(waitUntilRepeatPauseTime)
 			continue
 		}
-		return nil
+		return
 	}
-	return fmt.Errorf("node %s timed out waiting for condition, latest block height was %d", n.Name, latestBlockHeight)
+	n.t.Errorf("node %s timed out waiting for condition, latest block height was %d", n.Name, latestBlockHeight)
```

Comment:
Not blocking this PR, but generally confused about direction to not return an error here to flag in calling test, vs logging an error.

Reply:
`Errorf` does not only log but also fails immediately.

The reason for changing this is to reduce verbosity and simplify tests.

For the majority of these methods, we fail if the error is non-nil anyways. However, constantly doing the error checks reduces the readability of the tests.

Failing immediately via this `testing` package method reduces verbosity and makes it easier to write tests